.github/workflows/unittest.yml

```
@@ -0,0 +1,24 @@
+name: run python tests
+on: [push, pull_request]
+
+jobs:
+  python_unittests:
+      runs-on: ubuntu-latest
+      strategy:
+        matrix:
+          python: [2.7, 3.6]
```

Why not test for a couple more Python 3 versions too?

```suggestion
          python: [2.7, 3.6, 3.7, 3.8, 3.9]
```

---

we don't test those anywhere else?

---

But it's trivial to test them here, so why not?